content/docs/user-guide/packaging/windows-release-builds.md

```diff
@@ -0,0 +1,124 @@
+- Run your project's Game Launcher.
+
+
+## Prerequisites
+
+The following instructions assume that you have:
+
+- Set up O3DE on your computer. For help, refer to [Set up Open 3D Engine](\docs\welcome-guide\setup\_index.md).
+
+- Created an O3DE project that contains at least one level. You may need to resolve any errors in your project in order to create a release build.
+
+- Generated a Visual Studio project for your O3DE project. For help, refer to [Create a Visual Studio project](/docs/welcome-guide/create/creating-projects-using-cli/#create-a-visual-studio-project).
+
+This tutorial uses the following project name and directory in the examples.
+
+- Project name and location: `C:\MyProject`
+
+
+## Set the starting level
+
+Before you create a project game release layout, specify the starting level that the Game Launcher will load first.
+
+1. Open the `autoexec.cfg` file in your project's directory and add the following line. This sets the starting level by using `Loadlevel` and providing a path to the level. The path must be relative to the `Cache/pc` directory. In this example, the level asset is `mainmenu.spawnable`.
+
+    ```
+    Loadlevel levels/mainmenu/mainmenu.spawnable
+    ```
+
+## Process assets
+
+The Asset Processor or Asset Processor Batch are responsible for processing source assets into product assets, which the Game Launcher consumes at runtime. 
+
+1. Process your project's assets by following either one of the following instructions.
+    
+    - Use **CMake** to build the Asset Processor Batch and its dependencies by specifying the `MyProject.Assets` target. After building, this command runs Asset Processor Batch and processes the assets.
+
+        ```cmd
+        cmake --build build/windows_vs2019 --target MyProject.Assets --config profile -- /m
+        ```
+
+    - Run your project's Asset Processor to begin processing the assets.
+
+<!-- [Question] Is it accurate to say that both methods can complete this task? Is one method recommended? If so, why? -->
+
+{{< caution >}} Ensure that the Asset Processor or Asset Processor Batch processes all of your assets without failure. {{< /caution >}}
+
+
+## Create a project game release layout
+
+You can create a project game release layout by using a non-monolithic or monolithic build. A monolithic release layout only contains files that're necessary to run the project, such as the Game Launcher, packaged assets, and dependent `.dll` files. A non-monolithic release layout contains the same files needed to run the project, as well as the objects that you use for development such as tools, shader compilers, and all `.dll` files.
+
+In this step, you will use CMake to build either a monolithic or a non-monolithic project and specify the `INSTALL` target and `release` configuration. This handles several tasks:
+
+- Creates an `install` directory that will contain your release layout.
+
+- Packages an `engine.pak` file out of the product assets in your `Cache/pc` directory. 
```

Comment:
maybe we should mention here that this pak is non-optimized, that it will contain all assets, a better version is being worked on that will just pak the assets that are needed

Reply:
+1 and we should mention, if this file is larger than 4GiB it won't load in the game.